src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -174,34 +200,97 @@ CreateSpaceProcessor::pickHosts(PartitionID partId,
     return pickedHosts;
 }
 
-std::vector<HostAddr>
-CreateSpaceProcessor::pickHostsWithZone(PartitionID partId,
-                                        const std::vector<std::string>& zones,
-                                        int32_t replicaFactor) {
-    auto startIndex = partId;
-    std::vector<HostAddr> pickedHosts;
-    for (int32_t i = 0; i < replicaFactor; i++) {
-        auto zoneName = zones[startIndex++ % zones.size()];
-        auto zoneIdRet = getZoneId(zoneName);
-        if (!zoneIdRet.ok()) {
-            LOG(ERROR) << "Zone " << zoneName << " not found";
-            handleErrorCode(cpp2::ErrorCode::E_NOT_FOUND);
-            onFinished();
-            return pickedHosts;
+StatusOr<std::unordered_map<HostAddr, int32_t>>
+CreateSpaceProcessor::getHostLoading() {
+    std::unique_ptr<kvstore::KVIterator> iter;
+    const auto& prefix = MetaServiceUtils::partPrefix();
```

Comment:
do we need to get all parts of space?

Reply:
I will calculation the loading of each host